src/initial-submission/components/DetailsForm.tsx

```diff
@@ -48,13 +61,16 @@ const DetailsForm = ({ initialValues, ButtonComponent }: Props): JSX.Element =>
     //             .required(t('details.validation.subjects-required')),
     //     }),
     // });
+    let details = defaultManuscriptDetails(initialValues.manuscriptDetails);
+    details = useSuggestions(details, initialValues.suggestions || []);
```

Comment:
details does not need to be re set here, it can be set inside of the useSuggestions and we can keep it as a const.